packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -87,6 +87,25 @@ You can also add the Lambda to the Pipeline directly:
 fn.addToPipeline(lambdaStage, 'Lambda');
 ```
 
+The Lambda Action can have up to 5 inputs,
+and up to 5 outputs:
+
+```typescript
+const lambdaAction = fn.addToPipeline(lambdaStage, 'Lambda', {
+  inputArtifacts: [
+    sourceAction.outputArtifact,
+    buildAction.outputArtifact,
+  ],
+  outputArtifactNames: [
+    'Out1',
+    'Out2',
+  ],
+});
+
+lambdaAction.outputArtifacts(); // returns the list of output Artifacts
```

Comment:
I didn't make it a property, because similar methods in the CodeBuild Actions were not modeled as properties ([[1]](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-codebuild/lib/pipeline-actions.ts#L89), [[2]](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-codebuild/lib/pipeline-actions.ts#L173)), and I wanted to be consistent.

Reply:
+1 for consistency.

Reply:
As in... leave this as it is in this PR, or change the CodeBuild ones to properties? 😀

Reply:
OK, I'll leave it as a method then. This is probably something we should decide on more globally, so it's uniform across all our L2s.